### PR TITLE
[gardening] Fix 67 recently introduced typos

### PR DESCRIPTION
**Pull request by practicalswift**
Fix 67 recently introduced typos

### CODE REVIEW

File: validation-test/stdlib/NSNumberBridging.swift.gyb [hunk 1]
```diff
@@ -107,7 +107,7 @@ NSNumberTests.test("isTypePreservingNSNumber(_:)") {
 }
 
 NSNumberTests.test("_SwiftTypePreservingNSNumber.classForCoder") {
-  // Check that internal subclass is not achived.
+  // Check that internal subclass is not achieved.
```
Comment (gribozavr) [inline validation-test/stdlib/NSNumberBridging.swift.gyb]: It was meant to be 'archived'.

**Comment by gribozavr** [comment]
@swift-ci Please smoke test

File: validation-test/stdlib/NSNumberBridging.swift.gyb (hunk 1 above)
Comment (practicalswift) [inline validation-test/stdlib/NSNumberBridging.swift.gyb]: @gribozavr Thanks! Fixed!

**Comment by practicalswift** [comment]
@swift-ci please smoke test

**Pull request merged**